enricher/fabric8/src/main/java/io/fabric8/maven/enricher/fabric8/CamelRestDSLEnricher.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2016 Red Hat, Inc.
+ *
+ * Red Hat licenses this file to you under the Apache License, version
+ * 2.0 (the "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package io.fabric8.maven.enricher.fabric8;
+
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.Map;
+
+import io.fabric8.maven.core.util.Configs;
+import io.fabric8.maven.core.util.MapUtil;
+import io.fabric8.maven.enricher.api.BaseEnricher;
+import io.fabric8.maven.enricher.api.EnricherContext;
+import io.fabric8.maven.enricher.api.Kind;
+
+public class CamelRestDSLEnricher extends BaseEnricher {
+    static final String ENRICHER_NAME        = "f8-camel-dsl";
+
+    //Default Domain for Service Annotations
+    static final String DEFAULT_DOMAIN       = "api.service.kubernetes.io";
+
+    //Kubernetes Service Annotations
+    static final String SCHEME               = "scheme";
+    static final String PROTOCOL             = "protocol";
+    static final String PATH                 = "path";
+    static final String DESCRIPTION_LANGUAGE = "description-language";
+    static final String DESCRIPTION_PATH     = "description-path";
+
+    private File springConfigDir;
+    private String domain;
+
+    private enum Config implements Configs.Key {
+        springDir,
+        annotationsDomain;
+  
```

Comment:
@rhuss The user can provide a service.yml fragment to override values if needed. That seems sufficient to me.

Reply:
I mean comment to the code.

Reply:
> @rhuss The user can provide a service.yml fragment to override values if needed. That seems sufficient to me.

But its different then for other enrichers. Not everyone want to just write a fragment because he then would also know the keys of the annotations. Its really simpler to add it here, too. + it doesn't hurt and follows the other enrichers.

Reply:
@KurtStam if you don't want to change it yourself, I can update your PR if you like.